test

```diff
@@ -21,6 +21,7 @@ features=(
     "std perf-dfa"
     "std perf-inline"
     "std perf-literal"
+    "std pattern"
```

Comment:
Unfortunately, this script is only run on stable Rust. So I think you'll have to add

```
cargo test --test default --no-default-features --features 'std pattern'
```

as an explicit step in CI on nightly targets only.

Reply:
done